### PR TITLE
Users/sachinma/vmss bug fix

### PR DESCRIPTION
**Pull request by sachinma**

### CODE REVIEW

File: Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export default class VirtualMachineScaleSet {
 
             // if extension already exists, remove it
             if (!!matchingExtension) {
-                await this._deleteCustomScriptExtension(client, resourceGroupName, matchingExtension);
```
Comment (bishal-pdMSFT) [inline Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts]: This function can be removed from code

File: Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts [hunk 2]
```diff
@@ -98,7 +98,7 @@ export default class VirtualMachineScaleSet {
 
             // if extension already exists, remove it
```
Comment (bishal-pdMSFT) [inline Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts]: comment can be changed

File: Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts (hunk 1 above)
Comment (sachinma) [reply Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts]: Done

File: Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts (hunk 2 above)
Comment (sachinma) [reply Tasks/AzureVmssDeploymentV0/operations/VirtualMachineScaleSet.ts]: Done.

**Pull request merged**